### ISSUE TITLE
tabindex on timepicker does not work

### ISSUE THREAD

**Issue by getvictor** [angular-ui/bootstrap#4569]
When specifying **tabindex** on timepicker without spinners

`<uib-timepicker show-spinners="false" tabindex="8" />`

the surrounding table around timepicker is highlighted when getting to the corresponding tabindex, and not the individual inputs. Using Chrome on Mac.

When tabindex is specified, I'd like to see this same tabindex applied to each of the individual input fields of timepicker.

**Comment by wesleycho** [comment]
Just for clarity, what would your preferred solution be? Would it be for the individual inputs to also inherit the same tabindex (i.e. 8 in this example)?

**Comment by getvictor** [comment]
@wesleycho Yes, all the individual inputs should get the same tabindex. Elements with the same tabindex are processed in the order they appear. The outer element should not have a tabindex.

Reference: http://www.w3.org/TR/WCAG20-TECHS/H4.html#H4-ex3

**Comment by wesleycho** [comment]
This means that the tabindex should also be on the links as well.

**Comment by getvictor** [comment]
I'm not using the up/down links, which is what I believe you're referring
to. But yes, for accessibility purposes they should be on those links as
well.
